code/game/machinery/computer/communications.dm

```diff
@@ -456,7 +456,8 @@
 					dat += "<BR>\[ <A HREF='?src=\ref[src];operation=announce'>Make a Captain's Announcement</A> \]"
 					if(config.cross_allowed)
 						dat += "<BR>\[ <A HREF='?src=\ref[src];operation=crossserver'>Send a message to an allied station</A> \]"
-					dat += "<BR>\[ <A HREF='?src=\ref[src];operation=purchase_menu'>Purchase Shuttle</A> \]"
+					if(SSmapping.config.allow_custom_shuttles == "yes")

```

Comment:
Won't this be overwritten when nextmap.JSON is created?

Reply:
By the time that occurs shuttle purchase is already disabled because you're about to dock at centcomm.

Reply:
SSmapping.config never contains the next map's config. That's SSmapping.next_map_config